sso-tools/src/main/java/edu/mit/ll/nics/sso/util/SSOUtil.java

```diff
@@ -263,6 +263,18 @@ public String createUser(String email, String pass, String first, String last, b
 		return "{\"status\":\"" + ((val) ? "success" : "fail") + "\",\"message\":\"" + message + "\"}";
 	}
 
+    public String deleteUser(String uid) {
+        boolean status = false;
+        try {
+          status = utils.deleteUserIdentity(uid);
+        } catch(Exception e){
+            log.error("Error deleting user identity by uid : " + uid, e);
+        }
+        String message = status ? "successfully deleted identity with uid : " + uid : "failed to delete identity with uid : " + uid;
+
+        return "{\"status\":\"" + ((status) ? "success" : "fail") + "\",\"message\":\"" + message + "\"}";
```

Comment:
I see how you were following an existing pattern here and I appreciate that. Returning a string that can be parsed into JSON is not the kind of design we want to propagate in the project.